### PR TITLE
#141291017 wallet fixes

### PR DESCRIPTION
**Pull request by puritybirir**
#### What does this PR do?
It integrates wallet functionality and funds can be transferred from one consumer to the other.
 #### Description of Task to be completed?

 #### How should this be manually tested?

 #### What are the relevant pivotal tracker stories?
#141291017

 #### Screenshots (if appropriate)

### CODE REVIEW

File: imports/plugins/core/layout/client/templates/layout/wallet/wallet.html [hunk 1]
```diff
@@ -40,68 +20,20 @@ <h3 class="panel-title">Deposit</h3>
...
-            </div>
-            <div class="col-xs-12">
-              <input id="recipient" class="form-control" type="text" name="email" placeholder="Beneficiary Email">
-            </div>
-            <button type="submit" class="btn btn-success col-xs-2 col-md-offset-5" style="margin-top:15px">Transfer</button>
-          </form>
-        </div>
-      </div>
-    </div>
+    <script>
```
Comment (gangachris) [inline imports/plugins/core/layout/client/templates/layout/wallet/wallet.html]: This script doesn't seem to do anything

File: imports/plugins/core/layout/client/templates/layout/wallet/wallet.html [hunk 2]
```diff
@@ -40,68 +20,20 @@ <h3 class="panel-title">Deposit</h3>
...
+      function getAmount(){
+      var amount = document.getElementById('depositAmount').value
+      console.log(amount)
+      }
+    </script>
```
Comment (gngchrs) [inline imports/plugins/core/layout/client/templates/layout/wallet/wallet.html:36]: remove: Above script doesn't seem to do anything valuable

File: imports/plugins/core/layout/client/templates/layout/wallet/wallet.js [hunk 3]
```diff
@@ -1,209 +1,70 @@
...
-    if (res) {
-      document.getElementById("depositAmount").value = "";
-      Alerts.toast("Your deposit was successful", "success");
-    } else {
-      Alerts.toast("An error occured, please try again", "error");
+const payWithPaystack = amount => {
+  const userAccountInfo = Accounts.find(Meteor.userId()).fetch();
+  const emailAddress = userAccountInfo[0].emails[0].address;
+  const handler = PaystackPop.setup({
+    key: "pk_test_6860a0a85e1bbbb277c347e3e12e7f4373c8ba8c",
```
Comment (gngchrs) [inline imports/plugins/core/layout/client/templates/layout/wallet/wallet.js:29]: minor: shouldn't keys be set in the config file?

File: imports/plugins/core/layout/client/templates/layout/wallet/wallet.js [hunk 4]
```diff
@@ -1,209 +1,70 @@
...
-          });
-        }, (dismiss) => {
-          return dismiss === "cancel" ? false : true;
-        });
-      } else {
-        Alerts.toast("Invalid Pin", "error");
-      }
-    }
-    return true;
+    const amount = document.getElementById("depositAmount").value;
```
Comment (gngchrs) [inline imports/plugins/core/layout/client/templates/layout/wallet/wallet.js:67]: minor: validate amount. What happens when I pass in "ganga"

File: server/startup/accounts.js [hunk 5]
```diff
@@ -108,17 +108,17 @@ export default function () {
...
-          user.profile.account = 'Admin'
+          user.profile.account = "Admin";
         } else {
           Meteor.call('user/getCurrentUser', (err, res) => {
-          if (err) return alert(err)
-          const current_user = res;
-          user.profile.account = current_user.profile.account
-          return user.profile.account
-        });
+            if (err) return alert(err);
```
Comment (gngchrs) [inline server/startup/accounts.js:114]: minor: We are using alert?. 🤷‍♂️

**Pull request closed without merging**